### DISCUSSION TITLE
Extending tasmota with WebSerial?

### DISCUSSION THREAD

**Discussion by philfleck** [arendst/Tasmota#17926] [Q&A]
Hi, i am a long time Tasmota user, from the early days.
We are running a custom ESP32 platform, using the webserial platform to log into a computer plugged in via usb / uart (esp32-uart - usb-c-pc).

I could not find anything related, but is there way to nicely add this to tasmota?

https://randomnerdtutorials.com/esp32-webserial-library/

this might also be useful to others, the main usecase is a serial to hard to access devices over a local wifi (esp32) fe to reconfigure networkcards ...

best -g

**Comment by Jason2866** [comment]
There is the [TCP serial bridge ](https://tasmota.github.io/docs/Serial-to-TCP-Bridge/)function in Tasmota. Fully transparent to the data. [See](https://tasmota.github.io/docs/Zigbee/) as example
